query_optimizer/logical/Aggregate.cpp

```diff
@@ -72,6 +75,35 @@ std::vector<E::AttributeReferencePtr> Aggregate::getReferencedAttributes() const
   return referenced_attributes;
 }
 
+LogicalPtr Aggregate::copyWithNewInputExpressions(
+      const std::vector<E::ExpressionPtr>& input_expressions) const {
```

Comment:
What do you think about the reference sigils standards in Quickstep? If it applies, it looks like `const std::vector<E::ExpressionPtr> &input_expressions`.

https://github.com/pivotalsoftware/quickstep/wiki/Quickstep-Style-Guide